### PR TITLE
Adding Bastakol coordinants

### PR DESCRIPTION
**Pull request by Bastakol**

### CODE REVIEW

**Comment by jess010** [comment]
hi @Bastakol need to add a reference to @githubteacher and text that indicates that you are closing the issue e.g. closes <hashtag issue number>.

File: pins/Bastakol.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Bastakol
+latitude: 47.5637
+longitude: 7.564
+---
```
Comment (hectorsector) [inline pins/Bastakol.json]: @Bastakol 
Great work so far! Your pin is in the `pins` directory, though. Just make sure you move it into the `_pins` folder (with an underscore `_`), and that should allow the automated tests to pass!

**Comment by brianamarie** [comment]
Hi @Bastakol! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
@Bastakol I'm going to merge now. Nice work!

**Pull request merged**